glibc/plan.sh

```diff
@@ -304,15 +300,25 @@ ethers: files
 rpc: files
 EOF
 
-    extract_src tzdata
+    
+    # Install timezone data. 
+    # We set --sysconfdir=$pkg_prefix/etc during our build, so we need to
+    # embed timezone data in this package.
+    # 
+    # zic /dev/null creates posix timezones without leapseconds
+    # zic leapseconds creates right timezones with leapseconds
+    # zic -d "$ZONEINFO" creates  posixrules file. We use New York because POSIX 
+    #   requires the daylight savings time rules to be in accordance with US rules.
+
+    extract_src tzdatak
```

Comment:
``extract_src tzdatak`` (with a 'k' at the end) causes the build to fail for me with:

```bash
mkdir: created directory '/hab/pkgs/core/glibc/2.29/20190704100831/lib/locale'
'../glibc-2.29/nscd/nscd.conf' -> '/hab/pkgs/core/glibc/2.29/20190704100831/etc/nscd.conf'
/src/glibc/plan.sh: line 342: /src/glibc/../tzdatak/plan.sh: No such file or directory
   glibc: Build time: 12m46s
   glibc: Exiting on error
```
However, the build succeeds when I restore ``extract_src tzdata``

If this was a mistake, can you please restore the original ``extract_src tzdata``

Reply:
Good catch,  one of these days I'll learn how to drive vim :joy_cat: 

I'll have this fixed in an upcoming update